src/Illuminate/Database/Query/Builder.php

```diff
@@ -1338,20 +1337,18 @@ public function value($column)
      * Execute the query and get the first result.
      *
      * @param  array   $columns
-     * @return mixed|static
+     * @return \stdClass|array|null
```

Comment:
Collection doesn't extend stdClass. Did you mean `object`, or am I not reading this right?